Error: Schema version must be >= 7.4

## Steps to recreate:

1. Run the following commands:
```
git clone https://github.com/debootdevs/deboot.git --recurse-submodules 
cd deboot
sudo make init-env
```
2. Then run the following commands inside the virtual environment:
```
cd deboot
make BEE_VERSION=2.1.0 install
```
3. Observe the following error
```
[root@521bdb7fb3ea deboot]# make BEE_VERSION=2.0.0 install
make SYSROOT=/deboot/build/sysroot --directory appliance kiwi
make[1]: Entering directory '/deboot/appliance'
make SYSROOT=/deboot/build/sysroot --directory kiwi 
make[2]: Entering directory '/deboot/appliance/kiwi'
kiwi --profile Live system prepare --description . --root /deboot/build/sysroot && touch -h /deboot/build/sysroot/etc/os-release
[ INFO    ]: 21:34:10 | Reading runtime config file: '/etc/kiwi.yml'
[ INFO    ]: 21:34:10 | Loading XML description
[ INFO    ]: 21:34:10 | Support for multiple markup descriptions available
[ ERROR   ]: 21:34:10 | Unexpected error:
Traceback (most recent call last):
  File "/usr/bin/kiwi", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.12/site-packages/kiwi/kiwi.py", line 68, in main
    App()
  File "/usr/lib/python3.12/site-packages/kiwi/app.py", line 35, in __init__
    task_class().process()
  File "/usr/lib/python3.12/site-packages/kiwi/tasks/system_prepare.py", line 149, in process
    self.load_xml_description(
  File "/usr/lib/python3.12/site-packages/kiwi/tasks/base.py", line 179, in load_xml_description
    self.description = XMLDescription(
                       ^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/kiwi/xml_description.py", line 68, in __init__
    self.description = self.markup.get_xml_description()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/kiwi/markup/any.py", line 69, in get_xml_description
    return self.apply_xslt_stylesheets(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/kiwi/markup/base.py", line 87, in apply_xslt_stylesheets
    xslt_transform(parsed_description), pretty_print=True
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "src/lxml/xslt.pxi", line 603, in lxml.etree.XSLT.__call__
lxml.etree.XSLTApplyError: Error: Schema version must be >= 7.4 For details how to update older schemas refer to: https://osinside.github.io/kiwi
            
make[2]: *** [Makefile:11: /deboot/build/sysroot/etc/os-release] Error 1
make[2]: Leaving directory '/deboot/appliance/kiwi'
make[1]: *** [Makefile:10: /deboot/build/sysroot/etc/os-release] Error 2
make[1]: Leaving directory '/deboot/appliance'
make: *** [Makefile:52: /deboot/build/sysroot/etc/os-release] Error 2
[root@521bdb7fb3ea deboot]# 
```

## Screenshot

![image](https://github.com/debootdevs/deboot/assets/2212651/7e0cb9ad-031b-4933-a0eb-b1a756062999)

--- a/appliance/kiwi/config.xml.j2
+++ b/appliance/kiwi/config.xml.j2
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<image schemaversion="6.8" name="Fedora-38.0_appliance">
+<image schemaversion="7.4" name="Fedora-38.0_appliance">
     <description type="system">
         <author>Andrew W. Macpherson</author>
         <contact>mail@awmacpherson.com</contact>

